src/lib/storage/index/partial_hash/partial_hash_index_impl.cpp

```diff
@@ -128,13 +128,15 @@ size_t PartialHashIndexImpl<DataType>::estimate_memory_usage() const {
   bytes += sizeof(ChunkID) * _indexed_chunk_ids.size();
 
   bytes += sizeof(_positions);
+  bytes += sizeof(_null_positions);
+
+  // Since we do not exactly how much memory the hashmap has allocated at any given time, we use its .size() method to
```

Comment:
`Since we do not exactly know how much memory the hashmap has allocated` (missing know)